electrode-ota-server-routes-acquisition/test/is-protected-test.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from "chai";
+import isProtected from "../src/is-protected";
+
+describe("isProtected() Test", function() {
+    this.timeout(500000);
+    const dKey = "foo-bar-eeWWW-lorem-IpsiUM-FoO-bAr";
+
+    it("should return false by default", () => {
+        expect(isProtected()).to.be.equal(false);
+    });
+
+    it("should return false if no protectedPacks", () => {
+        expect(isProtected(dKey)).to.be.equal(false);
+    });
+
+    it("should return false if protectedPacks value is invalid", () => {
+        expect(isProtected(dKey, "")).to.be.equal(false);
+        expect(isProtected(dKey, " ")).to.be.equal(false);
+    });
+
+    it("should return false if the deployment key is not protected", () => {
+        const protectedPacks = "foo-bar-eeWWW-lorem-IpsiUM-FoO-bXr,foo-bar-eeWWW-lorem-IpsiUM-FoO-bYr,foo-bar-blah-blah-meh";
+        expect(isProtected(dKey, protectedPacks)).to.be.equal(false);
+    });
+
+    it("should return true if deploymentKey is protected", () => {
+        const protectedPacks = "foo-bar-eeWWW-lorem-IpsiUM-FoO-bZr,foo-bar-eeWWW-lorem-IpsiUM-FoO-bAr,foo-bar-blah-blah-meh";
+        expect(isProtected(dKey, protectedPacks)).to.be.equal(true);
+    });
+
+    it("deploymentKey should be case-sensitive", () => {
+        let protectedPacks = "foo-bar,blah-blah-meh-blah,f0o-Blah-bAr-m3h";
+        expect(isProtected("Foo-Bar", protectedPacks)).to.be.equal(false);
+        expect(isProtected("f0o-Blah-bAr-m3h", protectedPacks)).to.be.equal(true);
+    });
```

Comment:
add another test for `"foo-bar, blah-blah-meh-blah, f0o-Blah-bAr-m3h"`

Reply:
added

Reply:
Add
` expect(isProtected("foo-bar", "foo-bar, blah-blah-meh-blah, f0o-Blah-bAr-m3h")).to.be.equal(true);`

Reply:
added the same

Reply:
where did u add
expect(isProtected("foo-bar", "foo-bar, blah-blah-meh-blah, f0o-Blah-bAr-m3h")).to.be.equal(true);

Reply:
Added here: https://github.com/scrajesh/electrode-ota-server/blob/new-api/electrode-ota-server-routes-acquisition/test/is-protected-test.js#L38